app/assets/stylesheets/admin.scss

```diff
@@ -29,7 +29,7 @@ body > div.container {
 
 .users {
   li.user {
-    border-bottom: 1px solid $light-grey;
+    border-bottom: 1px solid $text-grey;
```

Comment:
Using a variable named text-gray for a border seems odd though, maybe just define `$text-gray` to `$light-gray` and `$light-gray` to the actual color in `colors.scss`?